QModManager/Utility/Dialog.cs

```diff
@@ -172,5 +173,62 @@ private IEnumerator ShowDialogEnumerator()
 
             confirmation.Select();
         }
+
+        private static void SetText(GameObject obj, string text)
+        {
+            if (SelectedTextType == null)
+            {
+                Type TxtType = typeof(Text);
+                Type TxtProType = Type.GetType("TMPro.TextMeshProUGUI, Unity.TextMeshPro", false, false);
+
+                if (TxtProType != null && obj.GetComponentInChildren(TxtProType) != null)
+                {
+                    SelectedTextType = TxtProType;
+                }
+                else if (TxtType != null && obj.GetComponentInChildren(TxtType))
+                {
+                    SelectedTextType = TxtType;
+                }
+                else
+                {
+                    Logger.Error("Unable to find Text component in dialog box");
+                    return;
+                }
+
+                textProperty = SelectedTextType.GetProperty("text", typeof(string));
+                fontSizeProperty = SelectedTextType.GetProperty("fontSize", typeof(float));
+            }
+
+            object txt = obj.GetComponentInChildren(SelectedTextType);
+
+            textProperty.SetValue(txt, text, null);
+        }
+
+        private static object[] ChangeAllFontSizes(GameObject obj, float change)
+        {
+            if (fontSizeProperty == null)
+                return null;
+
+            object[] textComponents = obj.GetComponentsInChildren(SelectedTextType);
+
+            ChangeAllFontSizes(textComponents, change);
+
+            return textComponents;
+        }
+
+        private static void ChangeAllFontSizes(object[] textComponents, float change)
+        {
+            if (fontSizeProperty == null)
+                return;
+
+            for (int i = 1; i < textComponents.Length; i++)
```

Comment:
Add comment to explain why this loop starts at 1.
```suggestion
            // Loop starts at 1 because text 0 is the main dialog text, which shouldn't be changed
            for (int i = 1; i < textComponents.Length; i++)
```